app/javascript/styles/mastodon/forms.scss

```diff
@@ -154,9 +154,10 @@ code {
       margin-bottom: 15px;
     }
 
-    li {
-      float: left;
-      width: 50%;
+    ul {
+      columns: 2;
+      -webkit-columns: 2;
```

Comment:
This and below is not necessary, we have autoprefixer generating these.

Reply:
Oops, my bad. I've updated this now.